README.md

```diff
@@ -18,6 +18,8 @@ Or install it yourself as:
 
     $ gem install word_2_quiz
 
+You will also need to install the [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
```

Comment:
This has been removed. We are now using antiword, and the requirement has been added to the readme.